resources/packs/azure/azure.lr

```diff
@@ -584,7 +584,7 @@ private azure.subscription.storageService.account.blobService.properties {
 }
 
 // Azure Storage Account Queue Service Properties
-private azure.subscription.storageService.account.queueService.properties {
+private azure.subscription.storageService.account.queueService.properties @defaults("id hourMetrics minuteMetrics") {
```

Comment:
@chris-rock same comment here as `blobService` ... `hourMetrics and minuteMetrics` return a lot of data. I think it might be too much for a default. 🤔 

```suggestion
private azure.subscription.storageService.account.queueService.properties @defaults("id") {
```